docs-source/sample_workflow.rst

```diff
@@ -15,118 +15,109 @@ Sample Workflow
 Needed imports and Variables
 ============================
 
-.. code-block:: python
 
-    import sys
-    import pathlib 
-    import secrets
-    from getpass import getpass
-    import requests
-    from fossology import Fossology, fossology_token
-    from fossology.obj import Group, AccessLevel, TokenScope
-    from fossology.exceptions  import FossologyApiError
-
-    FOSSOLOGY_SERVER = "http://fossology/repo"
+>>> import sys
+>>> import pathlib 
+>>> import secrets
+>>> from getpass import getpass
+>>> import requests
+>>> from fossology import Fossology, fossology_token
+>>> from fossology.obj import Group, AccessLevel, TokenScope
+>>> from fossology.exceptions  import FossologyApiError
+>>> FOSSOLOGY_SERVER = "http://fossology/repo"
 
 Create JWT Token
 ================
 
 Create token - if not already done - and store it in local file for reuse.
 
-.. code-block:: python
-
-  path_to_token_file = pathlib.Path.cwd() / '.token'
-  if not path_to_token_file.exists():
-    print("Enter your Fossology credentials, e.g. in the test environment 'username: fossy' and 'password: fossy'")
-    username = input("username: ")
-    pw = getpass()
-    token = fossology_token(
-        FOSSOLOGY_SERVER,
-        username,
-        pw,
-        secrets.token_urlsafe(8), # TOKEN_NAME seen in the database
-        TokenScope.WRITE,
-    )
-    with open(path_to_token_file, "w") as fp:
-        fp.write(token)
-    print("token written to .token")
-  else:
-    print("Reuse token from .token")
-    # Load the token
-    with open(".token", "r") as fp:
-        token = fp.read()
+
+>>> path_to_token_file = pathlib.Path.cwd() / '.token'
+>>> if not path_to_token_file.exists():
+...   #print("Enter your Fossology credentials, e.g. in the test environment 'username: fossy' and 'password: fossy'")
```

Comment:
Adding some comments here to explain why the `input()` method is not called here could be helpful.

Reply:
@thsetz please resolve the discussion when you commit new changes.